jax/experimental/jax2tf/saved_model_lib.py

```diff
@@ -21,47 +21,32 @@
 
 from typing import Callable, Sequence, Optional
 
-from jax.experimental import jax2tf # type: ignore[import]
+from jax.experimental import jax2tf  # type: ignore[import]
 import tensorflow as tf  # type: ignore[import]
 
 
-def save_model(jax_fn: Callable,
-               params,
-               model_dir: str,
-               *,
-               input_signatures: Sequence[tf.TensorSpec],
-               shape_polymorphic_input_spec: Optional[str] = None,
-               with_gradient: bool = False,
-               enable_xla: bool = True,
-               compile_model: bool = True):
-  """Saves the SavedModel for a function.
+def convert_and_save_model(jax_fn: Callable,
```

Comment:
We can make the definition here more concrete, for example we require a callable taking two args `Callable[[Any, Any], Any]`.

Reply:
This also makes me wonder, do we need to support models that want randomness as part of their inference (e,g, `predict_fn(params, rng, x)`)? Are there other signatures we might need to support in the future? It feels to me like the scope of this function might grow..

Reply:
I fixed the type hint for `jax_fn`. 

You are right that the scope of this function will grow. Or, we can decide that this function is just a convenience for the simple cases. For the more complicated cases, you should roll your own SavedModel.